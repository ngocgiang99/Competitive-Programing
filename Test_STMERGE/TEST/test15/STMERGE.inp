4
5 7
563028460 481488992 522112608 820538591 636710712 586712005 954686637 
257698045 18495995 187029687 399476194 356618211 80747816 862140602 
380622101 831337270 347483125 622772629 697635226 927679194 209096663 
853226316 90295046 881100533 380998787 921023012 577574703 599537644 
506323438 890491555 214923817 367448675 945155661 269616012 794567754 
2 2
541705702 120434726 
40033704 579308026 
1 6
489593599 836639474 122623038 252842148 812671211 255816754 
6 4
788426576 302773040 67785938 230070603 
48123476 433024927 335984743 490580299 
819536988 188703605 81486539 778697156 
175888909 973719271 277324481 345202444 
215502411 348136041 264087888 364514743 
256229435 798522546 63168956 686497946 
