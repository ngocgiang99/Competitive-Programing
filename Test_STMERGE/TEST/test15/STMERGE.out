466794708
40033704
255816754
230070603
